File: Assignment08/Assignment8_1.c
#include<stdio.h>

void Display(int iValue)
{
    if(iValue<0)
    {
        iValue=-iValue;
    }
    
    for(int i=0;i<iValue;i++)
    {
        printf("*\t");
    }
    for(int i=0;i<iValue;i++)
    {
        printf("#\t");
    }
    
}

int main()
{
    int iNo=0;

    printf("Enter The Number\n");
    scanf("%d",&iNo);

    Display(iNo);
    

    return 0;
}

File: Python/Assignment8/Assignment8_1.py

i=1

def Display(No):
    global i

    if(i<=No):
        print("* ",end=" ")
        i+=1
        Display(No)
    
def main():
    no=int(input("Enter Number: "))

    Display(no)


if __name__=="__main__":
    main()
